modules/installation-configuration-parameters.adoc

```diff
@@ -32,9 +32,6 @@
 // * installing/installing_vmc/installing-vmc-user-infra.adoc
 // * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
 // * installing/installing_vmc/installing-restricted-networks-vmc-user-infra.adoc
-// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
-// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
-// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
```

Comment:
Worth noting that while this work is for IBM Power VS, I am now just noticing that https://github.com/openshift/openshift-docs/pull/60438 add these references back to the module. These references were originally removed with https://github.com/openshift/openshift-docs/pull/61165